server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/EchoMethodContract.scala

```diff
@@ -167,4 +154,15 @@ trait EchoMethodContract {
 
     assertThatJson(response).isEqualTo(RESPONSE_OBJECT_WITH_UNSUPPORTED_METHOD)
   }
+
+  private def getHeadersWith(authHeader: Header): Headers = {
```

Comment:
IMO we should rely on the Auth scheme in this test class, and not position the header ourself.

Reply:
If we do that, we can test only the Auth scheme provided.